{"name": "test_premise[login0-case_data0]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/Users/nali/AutoApiTest/config'", "trace": "request = <SubRequest 'login' for <Function test_premise[login0-case_data0]>>\n\n>   ???\n\n/Users/nali/AutoApiTest/conftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/Users/nali/ApiAutoTest/common/unit/api_send_check.py:28: in api_send_check\n    res = api_send.api_send(case_data, case_dict, case_path, relevance)\n/Users/nali/ApiAutoTest/common/unit/api_send.py:85: in api_send\n    host = conf_manage(cur_host, 'HOST')\n/Users/nali/ApiAutoTest/common/config_module/config_manage.py:22: in conf_manage\n    param = cf.read_config(name)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <common.config_module.config_read.Config object at 0x11dbf4c88>\nsection = 'HOST'\n\n    def read_config(self, section):\n        \"\"\"\n        以dict方式返回ini文件中option键值对\n        :param section: section\n        :return: option键值对\n        \"\"\"\n        cf = ConfigOveride()\n>       os.chdir(self.path)\nE       FileNotFoundError: [Errno 2] No such file or directory: '/Users/nali/AutoApiTest/config'\n\n/Users/nali/ApiAutoTest/common/config_module/config_read.py:84: FileNotFoundError"}, "attachments": [{"name": "log", "source": "55a0b339-cda5-4e14-b121-6a36c00a1cd1-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "c44db728-90e5-44fe-a6d5-5cb94c96c050-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "login", "value": "{'test_name': 'login_h5', 'info': '密码登录m站', 'is_run': None, 'host': '${HOST1}$', 'headers': '${headers1}$', 'address': '/passport/api/pwdLogin', 'http_type': 'https', 'request_type': 'post_cookie', 'parameter_type': 'data', 'parameter': {'passport': 17521200974, 'password': 'anqi666', 'passportType': 1, 'sign': 'F017A0CD674A8B7A4F52B56898D78E6A'}, 'timeout': 10, 'cookies': False, 'check': {'expected_result': None, 'check_type': 'no_check', 'expected_code': None}, 'file': False, 'global_relevance': None, 'relevance': None}"}, {"name": "case_data", "value": "{'is_run': False}"}], "start": 1617801448561, "stop": 1617801448561, "uuid": "5dbc0bf5-19c7-4d96-ab90-e34051948090", "historyId": "8b80e4e5cb6e1f79fbd726fe8709f3e1", "testCaseId": "277933a2c8f509de68d6dbbe8d033af8", "fullName": "tests.TestScript.configsAPI.test_configs.TestConfigs#test_premise", "labels": [{"name": "severity", "value": "blocker"}, {"name": "feature", "value": "configs"}, {"name": "tag", "value": "@pytest.mark.skipif(False, reason='该用例的用例等级为：blocker，不在本次运行级别['blocker', 'critical', 'normal', 'minor']内')"}, {"name": "parentSuite", "value": "tests.TestScript.configsAPI"}, {"name": "suite", "value": "test_configs"}, {"name": "subSuite", "value": "TestConfigs"}, {"name": "host", "value": "anqidediannao.local"}, {"name": "thread", "value": "86863-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.TestScript.configsAPI.test_configs"}]}